test/unit/user.test.js

```diff
@@ -0,0 +1,83 @@
+const request = require('supertest');
+const { app } = require('../../server');
+const { User } = require('../../app/models');
+const { invalid, required } = require('../../app/dictionary');
+const { SCHEMA_VALIDATION, EXIST_REGISTER } = require('../../app/errors');
+
+const mockUser = {
+  firstName: 'Cristian',
+  lastName: 'Vega',
+  password: 'myPassword',
+  email: 'cristian.vega@wolox.co'
+};
+
+const notAllowedEmail = 'cristian.vega@gmail.com';
+const wrongEmail = 'thisIsNotaEmail';
+const wronPassword = '$%%^&U*(IDSD';
+
+describe('User controller, Post /users', () => {
+  const makeRequest = data =>
+    request(app)
+      .post('/users')
+      .send(data);
+
+  describe('sign up', () => {
+    it('should create a new user', async () => {
+      const { status, body } = await makeRequest(mockUser);
+      expect(status).toBe(201);
+      expect(body).not.toHaveProperty('password');
+      expect(body).toMatchObject({});
+
+      const {
+        dataValues: { email, id }
+      } = await User.findOne({ where: { email: mockUser.email } });
+      expect(id).toBeGreaterThanOrEqual(1);
+      expect(email).toBe(mockUser.email);
+    });
+
+    it('should fail when email is already used', async () => {
+      await User.newUser(mockUser);
```

Comment:
the idea is not to use the model in the test files, in the test folder, there is a folder called `factory`, please check it and check how works `factory-girl`